sandbox.html

```diff
@@ -457,6 +463,39 @@
   </div>
 </template>
 
+<template data-action="setSelection">
```

Comment:
You could define this template once and reuse it for the different actions, like it's done e.g. with "common-show".